web/src/main/resources/batch/applicationContext-batch-schedule.xml

```diff
@@ -5,10 +5,14 @@
 	xsi:schemaLocation="
 		http://www.springframework.org/schema/task http://www.springframework.org/schema/task/spring-task.xsd
 		http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd">
- 	<task:scheduled-tasks scheduler="scheduler">
+
+    <bean id="batchConfiguration" class="com.navercorp.pinpoint.web.batch.BatchConfiguration"/>
+
+    <task:scheduled-tasks scheduler="scheduler">
     	<task:scheduled ref="batchJobLauncher" method="alarmJob" cron="0 0/3 * * * *" />
         <task:scheduled ref="batchJobLauncher" method="agentCountJob" cron="0 0 2 * * *" />
 		<task:scheduled ref="batchJobLauncher" method="flinkCheckJob" cron="0 0/10 * * * *" />
+        <task:scheduled ref="batchJobLauncher" method="cleanupInactiveAgentsJob" cron="#{batchConfiguration['cleanupInactiveAgentsCron']}"/>
 	</task:scheduled-tasks>
 	
 	<task:scheduler id="scheduler" pool-size="1"/>
```

Comment:
It would be better to change the scheduler's pull size to 4.

Reply:
I have changed pool-size to 4. thanks :)